charts/rasa-x/values.yaml

```
@@ -140,6 +140,17 @@ app:
   port: 5055
   # resources which app is required / allowed to use
   resources:
+  # envs is a dict containing app specific env variables
+  envs:
```

to stay consistent with the other `extraEnvs`
```suggestion
  # extraEnvs is a dict containing app specific env variables
  extraEnvs: []
```

---

I can see the consistency argument here.  On the flip side, the reason I initially went with just `envs` was because there were no other envs defined for the app yet so "extra" seemed superfluous.  That said, I'm fine with having consistency between deployments so I can change that to extraEnvs.